config/toggles.js

```diff
@@ -224,6 +224,12 @@ const shortTermToggles = {
 
   enableBullForBulkDelSubj: environmentVariableTrue(
     pe, 'ENABLE_BULL_FOR_BULK_DEL_SUBJ'),
+
+  enableBullForBulkPostEvents: environmentVariableTrue(
+    pe, 'ENABLE_BULL_FOR_BULK_POST_EVENTS'),
+
+  enableBullForCreateAuditEvents: environmentVariableTrue(
+    pe, 'ENABLE_BULL_FOR_CREATE_AUDIT_EVENTS'),
```

Comment:
If we're going to have a bunch of these, it would be more manageable to set it to use a single env var with a comma-separated list of jobs to enable, like we do for activity logging.